jako/distribute/DistributedScan.py

```diff
@@ -30,7 +30,6 @@ def __init__(self,
                  clear_session=True,
                  save_weights=True,
                  config='config.json'):
-
         '''Distributed version of talos.Scan() for the local machine.
```

Comment:
Have a look in `Scan()` there are now two new arguments in there so it should be added in here too.

Reply:
Ah, the other will be added only once I merge the current branch where the support for multi-input models live.

Reply:
What should be done from my end now? Should I add the new arguments right away?

Reply:
Sure you can add `multi_input=False` in there. The other is actually not affecting `Scan()` so that will be the only one.

Reply:
Update here once you have made the change and I'll resolve the comment.